lib/msf/core/db_manager/cred.rb

```diff
@@ -3,13 +3,14 @@ module Msf::DBManager::Cred
   def creds(opts)
     query = nil
     ::ActiveRecord::Base.connection_pool.with_connection {
-      wspace = Msf::Util::DBManager.process_opts_workspace(opts, framework)
-      search_term = opts.delete(:search_term)
       # If :id exists we're looking for a specific record, skip the other stuff
       if opts[:id].present?
         return Metasploit::Credential::Core.where(id: opts[:id])
```

Comment:
Should this use `find` rather than `where` since we are only searching on ID and to be consistent with other modules?

Reply:
This and the above were caused by a merge conflict. I basically had added the same logic in two different branches so they weren't consistent since one was just isolated to creds.